software-modules/internet/chromium/chromium.sh

```diff
@@ -20,12 +20,32 @@ TARGET_DIR="/run/initramfs/memory/system/huronOS/software/internet/"
 
 ## Install software
 apt update
-apt install --yes --no-install-recommends $NAME
+apt install --yes --no-install-recommends $NAME python-is-python3
 apt autoremove --yes
 
 ## Prepare final files
 cp ./$NAME.desktop /usr/share/applications/
 
+## Prepare bookmarks extension
+# The extension
+curl -o /usr/share/bookmark-bridge.crx -L https://github.com/huronOS/bookmark-bridge/releases/latest/download/bookmark-bridge.crx
+
+# The extension loader
+CHROMIUM_EXT_LOADER_PATH="/usr/share/chromium/extensions"
+mkdir -p $CHROMIUM_EXT_LOADER_PATH
+curl -o "$CHROMIUM_EXT_LOADER_PATH/cbfajpicdmpnnoijdmhdomaailmdaiim.json" https://raw.githubusercontent.com/huronOS/bookmark-bridge/development/chrome-extension-loader.json
```

Comment:
Can't we pack the JSON with the CRX in a ZIP for the Github Releases? This way is more reliable than using development branch I guess. 
Also, it could be possible to use a system flag if it's a development build to decide if using a release commit or a development one.

Reply:
I don't think that to be needed since it won't be expected for those files to even change unless truly necessary.

Unless you mean how reliable would be to get a file itself from raw instead of release, do you?

Reply:
> I don't think that to be needed since it won't be expected for those files to even change unless truly necessary.
> 
> Unless you mean how reliable would be to get a file itself from raw instead of release, do you?

Yeah, I get that it won't change unless truly necessary but in overall is not a good practice to retrieve things from development branch if this code will become stable at some point. I would prefer to retrieve from main.